module/x/peggy/client/cli/oracle_tx.go

```diff
@@ -0,0 +1,169 @@
+		Short: "Submit an eth event observed by an orchestrator",
+		Args:  cobra.ExactArgs(8),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			cosmosAddr := cliCtx.GetFromAddress()
+
+			ethChainID := args[0]
+			ethContractAddress := args[1]
+			nonce, err := parseNonce(args[2])
+			if err != nil {
+				return err
+			}
+			receiverAddr, err := sdk.AccAddressFromBech32(args[3])
+			if err != nil {
+				return sdkerrors.Wrap(err, "cosmos receiver")
+			}
+			amount, err := strconv.ParseInt(args[4], 10, 64)
+			if err != nil {
+				return sdkerrors.Wrap(err, "amount")
+			}
+			tokenSymbol := args[5]
+
+			// Make the message
+			tokenContractAddr := types.NewEthereumAddress(args[6])
+			ethSenderAddr := types.NewEthereumAddress(args[7])
+			msg := types.MsgCreateEthereumClaims{
+				EthereumChainID:       ethChainID,
+				BridgeContractAddress: types.NewEthereumAddress(ethContractAddress),
+				Orchestrator:          cosmosAddr,
+				Claims: []types.EthereumClaim{
+					types.EthereumBridgeDepositClaim{
+						Nonce:          nonce,
+						ERC20Token:     types.NewERC20Token(uint64(amount), tokenSymbol, tokenContractAddr),
+						EthereumSender: ethSenderAddr,
+						CosmosReceiver: receiverAddr,
+					},
+				},
+			}
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func CmdSendETHWithdrawalRequest(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "withdrawal [eth chain id] [eth contract address] [nonce]",
+		Short: "Submit an eth event observed by an orchestrator",
+		Args:  cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			inBuf := bufio.NewReader(cmd.InOrStdin())
+			txBldr := auth.NewTxBuilderFromCLI(inBuf).WithTxEncoder(utils.GetTxEncoder(cdc))
+			cosmosAddr := cliCtx.GetFromAddress()
+
+			ethChainID := args[0]
+			ethContractAddress := args[1]
+			nonce, err := parseNonce(args[2])
+			if err != nil {
+				return err
+			}
+			msg := types.MsgCreateEthereumClaims{
+				EthereumChainID:       ethChainID,
+				BridgeContractAddress: types.NewEthereumAddress(ethContractAddress),
+				Orchestrator:          cosmosAddr,
+				Claims: []types.EthereumClaim{
+					types.EthereumBridgeWithdrawalBatchClaim{
+						Nonce: nonce,
+					},
+				},
+			}
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+			return utils.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
+		},
+	}
+}
+
+func CmdSendETHMultiSigRequest(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "multisig-update [eth chain id] [eth contract address] [nonce]",
+		Short: "Submit an eth event observed by an orchestrator",
```

Comment:
can we make the description more explicit. I think this is for updating the multisig set but I'm not 100% sure which I would say means the description is too vague.

Reply:
Fair comments on the doc. There was definitively no priority to make this a good UX 🙈